index.html

```diff
@@ -699,9 +682,9 @@ <h4>
       <dfn>NDEF Signature</dfn> is defined [[NDEF-SIGNATURE]].
       Its <a>TYPE field</a> contains "`Sig`" (`0x53`, `0x69`, `0x67`) and its
       <a>PAYLOAD field</a> contains version, signature and a certificate chain.
-      In this version of the API, Web NFC only provides the raw byte content
-      of the payload (see this
-      <a href="https://github.com/w3c/web-nfc/issues/363">issue</a>).
+    </p>
+    <p class="issue" data-number="363">
+      Web NFC only provides the raw byte content of the payload at the moment.
```

Comment:
As currently spec'ed,

Reply:
I suggested it as a replacement for that. I think it is more clear

Reply:
Done.